Source/WebKit/WebKitSwift/TextAnimation/TextAnimationManager.swift

```diff
@@ -101,32 +101,36 @@ extension TextAnimationManager: UITextEffectView.ReplacementTextEffect.Delegate
             Self.logger.debug("Can't get text preview. Incorrect UITextEffectTextChunk subclass")
             return nil
         }
+
         guard let delegate = self.delegate else {
             Self.logger.debug("Can't obtain Targeted Preview. Missing delegate." )
             return nil
         }
+
         guard let preview = await delegate.targetedPreview(for: uuidChunk.uuid) else {
             Self.logger.debug("Could not generate a UITargetedPreview")
             return nil
         }
+
         return preview
     }
     
     public func replacementEffectDidComplete(_ effect: UITextEffectView.ReplacementTextEffect) {
         self.effectView.removeEffect(effect.id)
 
-        guard let (animationID, _) = chunkToEffect.first(where: { (_, value) in value == effect.id }) else {
+        guard let (animationID, _) = self.chunkToEffect.first(where: { (_, value) in value == effect.id }) else {
```

Comment:
Swift noob question, what's the difference between the new and old line?

Reply:
It’s the same as other languages; it means the variable is an instance member variable and not a local variable. This just makes it more consistent throughout the file